samples/Adventure/AdventureGrainInterfaces/Thing.cs

```diff
@@ -1,32 +1,30 @@
-using Orleans.Concurrency;
-
 namespace AdventureGrainInterfaces;
 
-[Immutable]
+[GenerateSerializer, Immutable]
 public record class Thing(
-    long Id,
-    string Name,
-    string Category,
-    long FoundIn,
-    List<string> Commands);
+    [property: Id(0)] long Id,
+    [property: Id(1)] string Name,
+    [property: Id(2)] string Category,
+    [property: Id(3)] long FoundIn,
+    [property: Id(4)] List<string> Commands);
```

Comment:
Id attributes aren't necessary on properties in the primary constructor by default

Reply:
removed